Code/Engine/Texture/DirectXTex/DirectXTexMipmaps.cpp

```diff
@@ -15,7 +15,7 @@
 
 #include "DirectXTexP.h"
 
-#include "filters.h"
+#include "Filters.h"
```

Comment:
Urgh, will we contribute those fixes back? Because this is going to get lost in the next update.

Reply:
If you tell me the where to contribute back to, I can do that.

Reply:
To the nice folks at Microsoft: https://github.com/Microsoft/DirectXTex

Reply:
We can't actually contribute this back. In the official repository the "filters.h" file has the correct casing (lower case). But in ezEngine it is checked in as "Filters.h".

Reply:
Good, then they fixed it already and we still had the old casing in our system.